homeassistant/components/vicare/strings.json

```diff
@@ -377,6 +377,17 @@
       "message": "Unable to deactivate ViCare program {program}"
     }
   },
+  "options": {
+    "flow_title": "{name} ({host})",
+    "step": {
+      "init": {
+        "description": "If you have an advance API plan (see https://developer.viessmann.com/start/pricing.html), the rate limit for API calls is doubled (3000 instead of 1500).",
```

Comment:
URLs should not be in translations, instead use a placeholder to set the URL.

We do this for two reasons:

1. It prevents a URL from being translated accidentally.
2. If the URL changes, it doesn't result in the invalidation of all translations.

Reply:
```suggestion
        "description": "If you have an advance API plan (see {api_plan_url}), the rate limit for API calls is doubled (3000 instead of 1500).",
```

Reply:
@frenck do you mind if I fix the existing url in line 6 as well or does this lead to unnecessary translation work?